ehrql/backends/emis.py

```diff
@@ -30,6 +34,22 @@ class EMISBackend(SQLBackend):
         ehrql.tables.beta.smoketest,
     ]
 
+    EMIS_ORG_HASH = os.environ.get("EMIS_ORGANISATION_HASH", "emis_organisation_hash")
+    HASHED_ORG_COLUMN = sqlalchemy.literal(
+        EMIS_ORG_HASH, type_=sqlalchemy.VARCHAR
+    ).label("hashed_organisation")
+
+    def modify_inline_table_args(self, columns, rows):
+        columns.append(
+            sqlalchemy.Column(
+                self.HASHED_ORG_COLUMN.name,
+                self.HASHED_ORG_COLUMN.type,
+                key=self.HASHED_ORG_COLUMN.key,
+            )
+        )
+        rows = tuple([tuple([*row, self.EMIS_ORG_HASH]) for row in rows])
```

Comment:
Because I can't resist code-golfing, you could write this slightly more simply as:
```oy
rows = tuple((*row, self.EMIS_ORG_HASH) for row in rows)
```